docs/book/products/catalog_promotions.rst

```diff
@@ -273,6 +275,28 @@ Catalog Promotion Actions configuration reference
 | ``percentage_discount``       | ``['amount' => $amountFloat]``                                     |
 +-------------------------------+--------------------------------------------------------------------+
 
+How the Catalog Promotions are applied?
+---------------------------------------
+
+The Catalog Promotion application process utilises `Doctrine events <https://www.doctrine-project.org/projects/doctrine-orm/en/latest/reference/events.html>`_.
+When a new Promotion is created there is an event ``postPersist()`` in `CatalogPromotionEventListener <https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/CoreBundle/EventListener/CatalogPromotionEventListener.php>`_
```

Comment:
In fact `postPersist()` is a **function** not an **event** (a function that is triggered on `postPersist` event 💃)